providers/universal-provider/src/constants/values.ts

```diff
@@ -0,0 +1,5 @@
+export const LOGGER = "error";
+
+export const RELAY_URL = "wss://relay.walletconnect.com";
+
+export const STORAGE = "@universal_provider";
```

Comment:
Would it make sense to use a standardised format for the storage key prefixing like we do for Core, Sign etc? https://github.com/WalletConnect/walletconnect-monorepo/blob/v2.0/packages/core/src/constants/core.ts#L5

Also in terms of future-proofing/versioning the storage for possible breaking changes